samples/nrf9160/lwm2m_carrier/prj.conf

```diff
@@ -9,9 +9,6 @@ CONFIG_LWM2M_CARRIER=y
 # The library requires newlibc
 CONFIG_NEWLIB_LIBC=y
 
-# The library needs a source of entropy
-CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
There are two `overlay-carrier.conf` files that should be updated as well in mqtt and coap samples.

Reply:
Fixed